### PR TITLE
pcapplusplus: add linux support and fix linkorders

### PR DESCRIPTION
**Pull request by Doekin**

### CODE REVIEW

File: packages/p/pcapplusplus/xmake.lua [hunk 1]
```diff
@@ -16,12 +16,17 @@ package("pcapplusplus")
...
+        add_deps("libpcap")
+    end
+
+    on_install("windows", "linux", function (package)
         local configs = {}
         table.insert(configs, "-DPCAPPP_BUILD_EXAMPLES=OFF")
         table.insert(configs, "-DPCAPPP_BUILD_TESTS=OFF")
         table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:is_debug() and "Debug" or "Release"))
         import("package.tools.cmake").install(package, configs)
+        package:add("linkorders", {"Pcap++", "Packet++", "Common++"})
```
Comment (waruqi) [inline packages/p/pcapplusplus/xmake.lua:29]: if only tree libs, you can use add_links to set them.

Comment (Doekin) [reply packages/p/pcapplusplus/xmake.lua:29]: 原先未设定链接顺序时，链接 flag 为 `-lPacket++ -lPcap++ -lCommon++`
会产生错误 ``` undefined reference to `pcpp::RawPacket::RawPacket()' ```
是否有更恰当的写法？

Comment (star-hengxing) [reply packages/p/pcapplusplus/xmake.lua:29]: 用 `add_links("Pcap++", "Packet++", "Common++")`

Comment (Doekin) [reply packages/p/pcapplusplus/xmake.lua]: OK，已修改

**Pull request merged**